public/locales/es/administration.json

```diff
@@ -108,7 +108,8 @@
             "button": "Eliminar administrador(a)",
             "confirmStart": "¿Estás seguro de que quieres eliminar a ",
             "confirmEnd": " como usario administrador?",
-            "cannotBeUndone": "Esta acción no se puede deshacer sin otro/a administrador(a)"
+            "cannotBeUndone": "Esta acción no se puede deshacer sin otro/a administrador(a)",
+            "lastAdmin": "Advertencia: Tienes que hacer otro usario un adminstrador para poder eliminar tu status de administrador(a)."
```

Comment:
should this be ¡Advertencia! Tienes....?

Reply:
I added the exclamation points to both the Spanish and English